components/usage/pkg/apiv1/billing.go

```diff
@@ -326,12 +326,76 @@ func (s *BillingService) ReconcileInvoices(ctx context.Context, in *v1.Reconcile
 		log.WithError(err).Errorf("Failed to udpate usage in stripe.")
 		return nil, status.Errorf(codes.Internal, "Failed to update usage in stripe")
 	}
+	err = s.ReconcileStripeCustomers(ctx)
```

Comment:
This is the only place where the new reconcilation logic is called. We only log the error.

Reply:
For me understanding - is `ReconcileInvoices` something we call periodically that also updates the usage on stripe's end? So this would attach this additional check to correct any customers as well?

Reply:
Yes, it regularly revisits the stripe customers and syncs gitpod <> stripe. 
I thought that somewhat fits and I could just piggy-back this control loop 😇